tests/char2.dora

```diff
@@ -1,8 +1,8 @@
 fun main() {
-    let x = Array::<Char>();
-    assert(x.len() == 0);
+    //let x = arrayEmpty::<Char>();
```

Comment:
I think this doesn't need to be commented out

Reply:
Fixed!